src/theme/foundations/colours.ts

```diff
@@ -1,4 +1,26 @@
 // Colour schemes available for isomer
+
+export type IsomerBaseColorScheme = "purple"
```

Comment:
maybe out of scope but can we shift `primary + secondary` here and rename to `IsomerColourPalettes`?

Reply:
This change will incur alot of file changes right? In separate PR uh

Reply:
Sorry, I'm not clear why it would incur alot of changes - could you go into more detail?

The theme that chakra reads from is passed by us as an object then chakra just reads the key we pass in, so as long as the final object contains the `primary.<palette>` key, nothing should change!